iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -6105,12 +6103,18 @@
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				BUNDLE_IDENTIFIER = app.omaps.debug;
 				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
+				CLANG_UNDEFINED_BEHAVIOR_SANITIZER_NULLABILITY = NO;
```

Comment:
Are you sure about this?

Reply:
No, I'm not certain.  My goal was to fix the most errors and re-force "zero-warning" policy. Unfortunately, I realized that clang's built sanitizer is too noisy. I agree that "nullability" checks are good in general.  Unfortunately, we don't have enough resources to resolve all problems of such kind. I think we need to preserve a status-quo and at least don't introduce new problems.

Reply:
Do not do it if you are not certain. Let's revert it.